modules/ROOT/pages/release-notes.adoc

```diff
@@ -17,4 +17,12 @@ Release notes for {productname} 6.0
 
 a|
 
```

Comment:
This isn't rendering correctly, as you need an even amount of cells. You have 3 now and the top right is empty when it should have 6.0.2. So in this case we probably should delete these lines.

- [ ] Fix table

Reply:
The other option is to change the `cols` so it only has a single column. However, this was setup intending to have 2 columns (like other pseudo index pages).

Reply:
Should hopefully be fixed now